### PR TITLE
LPS-113948 Add Search for Account Groups

### PR DESCRIPTION
**Pull request by alee8888**
https://issues.liferay.com/browse/LPS-113948

@jonmak08 @drewbrokke

### CODE REVIEW

**Comment by alee8888** [comment]
ci:test:sf

File: modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java [hunk 1]
```diff
@@ -54,6 +66,26 @@ public AccountGroup addAccountGroup(
...
+		long companyId, String keywords, int start, int end,
+		OrderByComparator obc) {
+
+		DynamicQuery accountGroupDynamicQuery = _getAccountGroupDynamicQuery(
+			companyId, keywords, obc);
+
+		if (accountGroupDynamicQuery == null) {
+			return new BaseModelSearchResult<>(
+				Collections.<AccountGroup>emptyList(), 0);
+		}
```
Comment (pei-jung) [inline modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupLocalServiceImpl.java:89]: This check is not necessary. `_getAccountGroupDynamicQuery` will never return null.

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:sf

**Comment by alee8888** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**